lib/tasks/settings.rake

```diff
@@ -5,3 +5,9 @@ namespace :settings do
     Setting.add_new_settings
   end
 end
+
+desc "Rename existing settings"
+task rename_setting_keys: :environment do
+  ApplicationLogger.new.info "Renaming existing settings"
+  Setting.rename_key from: "dashboard.emails",  to: "feature.dashboard.notification_emails"
+end
```

Comment:
I think this task needs to be inside the `namespace :settings do` block :thinking:. Right now I'm getting an error message:
```
$ rake settings:rename_setting_keys                                                                 ◼
rake aborted!
Don't know how to build task 'settings:rename_setting_keys' (See the list of available tasks with `rake --tasks`)
Did you mean?  settings:add_new_settings
```

Reply:
Nice catch @javierm 
I don't know how I ended up committing it out of the namespace. My fault 🙏 
Now it's solved. 
Thank you for review!